node/scripts/entrypoint.sh

```diff
@@ -0,0 +1,13 @@
+#! /bin/bash -x
+
+echo "Install iscsid..."
+yum install -y iscsi-initiator-utils && yum clean all
```

Comment:
I don't think worker node should require internet access. this should be at compilation time.

Reply:
removed